src/components/List.jsx

```diff
@@ -8,6 +14,26 @@ export default function List() {
   const [docs, setDocs] = useState([]);
   const [userToken] = useState(getUser());
 
+  const handleCheckBox = (e, item) => {
+    e.preventDefault();
+    let now = Timestamp.now().seconds;
+    const docItem = doc(db, userToken, item.id);
+    updateDoc(docItem, {
+      lastPurchaseDate: now,
+    });
+  };
+
+  const check24Hrs = (item) => {
+    let now = Timestamp.now().seconds;
+    let itemPurchaseDate = item.data().lastPurchaseDate;
+    let difference = now - itemPurchaseDate;
+    // We are using 86400 which is 24 hours in seconds since `now` is also in seconds
+    if (difference < 86400) {
+      return true;
+    }
+    return false;
```

Comment:
This is 100% valid syntax. If you want to reduce verbosity or bring this to one line we could simplify this since `difference < 86400` will return either true or false already`:

```suggestion
   return difference < 86400
```

Reply:
Thanks! We like that suggestion and have committed an update with that change.